com.jianguoyun.Nutstore.appdata.xml

```diff
@@ -21,7 +21,6 @@
       注意：本flatpak打包版本没有得到上海亦存网络科技有限公司的认证和支持。
     </p>
   </description>
-  <launchable type="desktop-id">com.jianguoyun.Nutstore.desktop</launchable>
```

Comment:
Any reason for removing this? It used to tell the Software center how to open the application

Reply:
You said "As they are already defined in the desktop file, they can be removed from here.", what should I remove here?

Reply:
I meant the categories